content/en/docs/components/pipelines/sdk/output-viewer.md

```diff
@@ -40,17 +40,17 @@ Pipelines UI:
       alt="Output visualization from a pipeline run"
       class="mt-3 mb-3 border border-info rounded">
 
-* The **Artifacts** tab shows the visualization for the selected pipeline step.
+* The **Visualizations** tab shows the visualization for the selected pipeline step.
```

Comment:
nit: you can also remove the Outdated banner in line 7

Reply:
Thank you Yuan, removing outdated banner after the updates for PR comments here.